common/components/common/selection/SelectorCollapsible.jsx

```diff
@@ -157,17 +157,17 @@ class SelectorCollapsible<T> extends React.PureComponent<Props<T>, State> {
     });
   }
   _renderSelectAll(category): React$Node {
+    //TODO: replace this absurdly hacky method of generating a subcategory/category name
+    //it works because we can rely on the _.groupBy() and filteredOptions in this.initializeOptions, idx 0 of any given cat/subcat is guaranteed to exist (no empty categories allowed) and return the right value due to grouping.
     let selectAllId = category[0].subcategory || category[0].category
     return (
       <span>
         <label className="CollapsibleMenuItem">
           <
             input type="checkbox"
             className="ContextualCollapsible-selectAll"
-            checked={this.state[selectAllId]}
+            checked={this._selectAllController.bind(this, category)}
```

Comment:
checked={} and onChange={} are where this all goes poorly. _handleSelectAll works more or less assuming I feed it the right conditional value to check (it's presently wrong) and the _selectAllController is uhhhh well I'd just delete it and start over